daemon/daemon-posix.cc

```diff
@@ -102,19 +89,19 @@ static bool setup_signals(void* arg)
 
 #else /* no signalfd API, use evsignal */
 
-static void reconfigure(evutil_socket_t /*fd*/, short /*events*/, void* /*arg*/)
+static void _reconfigure(evutil_socket_t /*fd*/, short /*events*/, void* arg)
```

Comment:
nit: https://en.cppreference.com/w/cpp/language/identifiers
Maybe it applies here.

Reply:
Like @nevack says, this may be a reserved name. Since it's inside the tr_daemon class, it's not a global scope so maybe it's not reserved? ...TBH I'm not sure.

To remove ambiguity, and for consistency with some of the other code, possibly rename this e.g. reconfigureMarshall() for consistency with the same static-to-object-method rechokePulseMarshall() in libtransmission/peer-mgr.cc